best-practices/server-side/framework/README.md

```diff
@@ -1,9 +1,19 @@
 
-- [Node](node/README.md) - Node.js® is a JavaScript runtime built on Chrome's V8 JavaScript engine.
-- [Python](python/README.md) - Python is a programming language that lets you work quickly and integrate systems more effectively.
-- [Ruby](ruby/README.md) - A dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write. 
-- [.NET](dotnet/README.md) - .NET is a free, cross-platform, open source developer platform for building many different types of applications.
-- [PHP](php/README.md) - PHP is a popular general-purpose scripting language that is especially suited to web development.
+* Javascript - JavaScript, often abbreviated as JS, is a programming language that conforms to the ECMAScript specification. JavaScript is high-level, often just-in-time compiled, and multi-paradigm. It has curly-bracket syntax, dynamic typing, prototype-based object-orientation, and first-class functions.
+  - [Node](node/README.md) - Node.js® is a JavaScript runtime built on Chrome's V8 JavaScript engine.
+  - [MEAN](/node/mean/README.md) - MEAN is a full-stack JavaScript solution that helps you build fast, robust, and maintainable production web applications using MongoDB, Express, AngularJS, and Node.js.
+
+* [Python](python/README.md) - Python is an interpreted, high-level, general-purpose programming language. Created by Guido van Rossum and first released in 1991, Python's design philosophy emphasizes code readability with its notable use of significant whitespace.
+  - [Django](python/django/README.md) - Django is a Python-based free and open-source web framework, which follows the model-template-view architectural pattern.
+  - Flask - Flask is a micro web framework written in Python. It is classified as a microframework because it does not require particular tools or libraries. It has no database abstraction layer, form validation, or any other components where pre-existing third-party libraries provide common functions.
+  
+* [Ruby](ruby/README.md) - A dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write. 
+  - Ruby On Rails - Ruby on Rails, or Rails, is a server-side web application framework written in Ruby under the MIT License. Rails is a model–view–controller framework, providing default structures for a database, a web service, and web pages.
+* C# - C# is a general-purpose, multi-paradigm programming language encompassing strong typing, lexically scoped, imperative, declarative, functional, generic, object-oriented, and component-oriented programming disciplines.
+  - [.NET](dotnet/README.md) - .NET is a free, cross-platform, open source developer platform for building many different types of applications.
+* [PHP](php/README.md) - PHP is a popular general-purpose scripting language that is especially suited to web development.
```

Comment:
I desagreed with "general-purpose". We can not create a desktop application with PHP. PHP is limited to Web applications. 
See this comparison: https://www.educba.com/php-vs-c-sharp/

Reply:
This can be change to : ```PHP is a popular scripting language...```

Reply:
All of these descriptions are directly lifted from wikipedia, which in turn is copied from the php website here: https://www.php.net/

I believe that we should keep with the official descriptions if at all possible, since that will not age as much as us trying to custom create our own.

Reply:
Well, I still disagree with "General-purpose". PHP in nature is limited to web environment. In my opinion, what is written is not always correct, even from official websites.